<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DiffView</class>
 <widget class="QMainWindow" name="DiffView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>900</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DiffView</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTextEdit" name="edit_diff">
      <property name="font">
       <font>
        <family>Monospace</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="lineWrapMode">
       <enum>QTextEdit::NoWrap</enum>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label_status">
        <property name="text">
         <string>&lt;status&gt;</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QCheckBox" name="check_hide_empty_meta">
        <property name="text">
         <string>Hide Empty Meta</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="button_hide_selection">
        <property name="text">
         <string>Hide Selection</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="button_overwrite_file">
        <property name="text">
         <string>Overwrite File</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>edit_diff</tabstop>
  <tabstop>button_hide_selection</tabstop>
  <tabstop>button_overwrite_file</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
